cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java

```diff
@@ -601,10 +601,14 @@ private void convertEquipmentAtBoundaryNode(Context context, String node) {
                 convertTwoEquipmentsAtBoundaryNode(context, node, connectedBeqs.get(0), connectedBeqs.get(1));
                 // Log ignored AcLineSegments
                 beqs.stream().filter(beq -> !connectedBeqs.contains(beq)).collect(Collectors.toList())
-                    .forEach(beq -> context.ignored("convertEquipmentAtBoundaryNode",
-                        String.format("Multiple AcLineSegments at boundary %s. Disconnected AcLineSegment %s is ignored", node, beq.getAcLineSegmentId())));
```

Comment:
Maybe add a comment explaining why we recorded the disconnected AcLineSegments as dangling lines.